### ISSUE TITLE
Consider dropping camel_case and no-long-files

### ISSUE THREAD

**Issue by Wilfred** [bahmutov/eslint-rules#14]
I believe these can now be replaced by [camelcase](http://eslint.org/docs/rules/camelcase) and [max-lines](http://eslint.org/docs/rules/max-lines) which are built-in to eslint.

**Comment by bahmutov** [comment]
yeah, good catch, I should remove these, hmm, when do I get time to do this?!
